meson.build

```diff
@@ -0,0 +1,33 @@
+project('gst-checksumsink', 'c',
+  version : '1.0',
+  meson_version : '>= 0.54',
+  default_options : [ 'warning_level=1',
+                      'buildtype=debugoptimized' ])
+
+gst_version = meson.project_version()
+version_arr = gst_version.split('.')
+gst_version_major = version_arr[0].to_int()
+gst_version_minor = version_arr[1].to_int()
+gst_req = '>= @0@.@1@.0'.format(gst_version_major, gst_version_minor)
```

Comment:
Isn't this basically just:

```meson
gst_req = '>= @0@.0'.format(meson.project_version())
```

Reply:
yes, basically

Reply:
...or basically just:
```python
gst_req = '>= @0@'.format(meson.project_version())
```
...however, it actually should be
```
gst_req = '>= 1.2`
```
... to be consistent with `configure.ac`